nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/test/groovy/org/apache/nifi/processors/elasticsearch/TestElasticSearchClientService.groovy

```diff
@@ -15,82 +15,77 @@
  * limitations under the License.
  */
 
-package org.apache.nifi.processors.elasticsearch;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.apache.nifi.controller.AbstractControllerService;
-import org.apache.nifi.elasticsearch.DeleteOperationResponse;
-import org.apache.nifi.elasticsearch.ElasticSearchClientService;
-import org.apache.nifi.elasticsearch.IndexOperationRequest;
-import org.apache.nifi.elasticsearch.IndexOperationResponse;
-import org.apache.nifi.elasticsearch.SearchResponse;
-
-import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-public class TestElasticSearchClientService extends AbstractControllerService implements ElasticSearchClientService {
-    private boolean returnAggs;
-    private boolean throwErrorInSearch;
-    private boolean throwErrorInDelete;
-
-    public TestElasticSearchClientService(boolean returnAggs) {
-        this.returnAggs = returnAggs;
+package org.apache.nifi.processors.elasticsearch
+
+import org.apache.nifi.controller.AbstractControllerService
+import org.apache.nifi.elasticsearch.*
```

Comment:
Although we don't currently have all the correct Checkstyle rules in place for Groovy files, we should still avoid star imports when we can.